tests/e2e/backup_restore_suite_test.go

```diff
@@ -104,6 +104,15 @@ var _ = Describe("AWS backup restore tests", func() {
 				Skip(reason)
 			}
 
+			if provider == "azure" && brCase.BackupRestoreType == CSI {
+				if brCase.MinK8SVersion == nil {
+					brCase.MinK8SVersion = &K8sVersion{Major: "1", Minor: "24"}
```

Comment:
version >= 4.10 should be min k8s version 1.23
```suggestion
					brCase.MinK8SVersion = &K8sVersion{Major: "1", Minor: "23"}
```